### PR TITLE
gh-143 styling assets top tab view; fixing currency formatter;

### LINKED ISSUE #143: Styling Coins/Collectibles top tab bar

### PR DESCRIPTION
**Pull request by sche**
closes #143

### CODE REVIEW

File: Multisig/UI/Assets/AssetsView.swift [hunk 1]
```diff
@@ -12,14 +12,33 @@ struct AssetsView: View {
...
+                if selectedSafe.first != nil {
+                    BalancesView(safe: selectedSafe.first!)
+                } else {
+                    AddSafeIntroView()
+                }
+            }
+            .gnoTabItem(id: 0) {
+                HStack {
+                    Image("ico-coins")
+                    Text("COINS").font(Font.gnoCaption1)
```
Comment (sche) [inline Multisig/UI/Assets/AssetsView.swift:29]: Here I could not use CaptionText() as it has 'foregroundColor' that overrides top tabs accent color for selected/not selected states

File: Multisig/UI/UI Library/Text/CaptionText.swift [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+//  CaptionText.swift
+//  Multisig
+//
+//  Created by Andrey Scherbovich on 20.05.20.
+//  Copyright © 2020 Gnosis Ltd. All rights reserved.
+//
+
+import SwiftUI
+
+struct CaptionText: View {
```
Comment (DmitryBespalov) [inline Multisig/UI/UI Library/Text/CaptionText.swift:11]: If it's not used, do we need it? 🤔

File: Multisig/UI/Assets/AssetsView.swift (hunk 1 above)
Comment (DmitryBespalov) [reply Multisig/UI/Assets/AssetsView.swift:29]: On `iPhone 8 Plus`, `iPhone 11 Pro (13.4.1)`, and `iPhone Pro Max`  the Coins text is truncated. (Ok on `iPhone SE 2nd`, `iPhone 6s`, `iPhone 8`, `iPhone 8 Plus`)

File: Multisig/UI/App/ContentView.swift [hunk 3]
```diff
@@ -30,7 +30,7 @@ struct ContentView: View {
 
                 BottomTabView($viewState.state) {
 
-                    TempAssets()
+                    AssetsView()
```
Comment (DmitryBespalov) [inline Multisig/UI/App/ContentView.swift:33]: Please rename this tab to the "Assets" as it is now in Zeplin

File: Multisig/UI/UI Library/Text/CaptionText.swift (hunk 2 above)
Comment (sche) [reply Multisig/UI/UI Library/Text/CaptionText.swift:11]: removed

File: Multisig/UI/App/ContentView.swift (hunk 3 above)
Comment (sche) [reply Multisig/UI/App/ContentView.swift:33]: ok

File: Multisig/UI/Assets/AssetsView.swift (hunk 1 above)
Comment (sche) [reply Multisig/UI/Assets/AssetsView.swift:29]: for me it works in simulators

File: Multisig/UI/Assets/AssetsView.swift [hunk 4]
```diff
@@ -12,14 +12,33 @@ struct AssetsView: View {
...
+                }
```
Comment (DmitryBespalov) [inline Multisig/UI/Assets/AssetsView.swift:30]:
```suggestion
                }
                .frame(maxWidth: .infinity)
```

File: Multisig/UI/Assets/AssetsView.swift [hunk 5]
```diff
@@ -12,14 +12,33 @@ struct AssetsView: View {
...
             }
+
+            Text("Coming soon")
+                .gnoTabItem(id: 1) {
+                    HStack {
+                        Image("ico-collectibles")
+                        Text("COLLECTIBLES").font(Font.gnoCaption1)
+                    }
```
Comment (DmitryBespalov) [inline Multisig/UI/Assets/AssetsView.swift:38]:
```suggestion
                }
                .frame(maxWidth: .infinity)
```

File: Multisig/UI/Assets/AssetsView.swift (hunk 1 above)
Comment (DmitryBespalov) [reply Multisig/UI/Assets/AssetsView.swift:29]: ok, see the code suggestions that fixes it

File: Multisig/UI/Assets/BalancesView.swift [hunk 6]
```diff
@@ -8,12 +8,12 @@
 
 import SwiftUI
 
-struct CoinsView: View {
+struct BalancesView: View {
```
Comment (DmitryBespalov) [inline Multisig/UI/Assets/BalancesView.swift:11]: The row height is too large

Screenshot 2020-05-20 at 14 53 13

File: Multisig/UI/Assets/AssetsView.swift (hunk 4 above)
Comment (sche) [reply Multisig/UI/Assets/AssetsView.swift:30]: funny that if you make the same for collectibles, it shrinks

File: Multisig/UI/Assets/BalancesView.swift (hunk 6 above)
Comment (sche) [reply Multisig/UI/Assets/BalancesView.swift:11]: done

**Pull request merged**